### DISCUSSION TITLE
Is there any way to fix the camera bounds when switching models

### DISCUSSION THREAD

**Discussion by PROJECT-SPARKS-LTD** [google/model-viewer#3563] [Q&A]
We are nearly complete on building a small product configurator demo but have become stuck this last hurdle.

I have a base product that loads on first view with several buttons that can swap in different versions of this product with additional attachments.

With the default Model Viewer behaviour it seems that on loading a larger model the camera reframes the scene using the new models larger bounding box.  Is there anyway to disable this functionality?  

When loading the new model I would like the camera to maintain its original zoom level even if this means the larger object is clipped by the edges of the viewer.

Mockup below:

![image](https://user-images.githubusercontent.com/103035806/176901899-9acbfa51-c82b-40c9-b6d4-dbf1d677de72.png)

**Comment by elalish** [comment]
Yes, simply use:
```
const orbit = mv.getCameraOrbit();
const fov = mv.getFieldOfView();
mv.src = "newModel.glb";
mv.cameraOrbit = orbit.toString();
mv.fieldOfView = `${fov}deg`;
```
https://modelviewer.dev/docs/#entrydocs-stagingandcameras-methods-getCameraOrbit

**Reply by PROJECT-SPARKS-LTD** [reply]
Brilliant! Thanks Emmett.  I needed to add the same update to the camera target as well and now its working perfectly.  Thanks for the help
